pandas/_typing.py

```diff
@@ -28,7 +29,7 @@
 AnyArrayLike = TypeVar("AnyArrayLike", "ExtensionArray", "Index", "Series", np.ndarray)
 ArrayLike = TypeVar("ArrayLike", "ExtensionArray", np.ndarray)
 DatetimeLikeScalar = TypeVar("DatetimeLikeScalar", "Period", "Timestamp", "Timedelta")
-Dtype = Union[str, np.dtype, "ExtensionDtype"]
+Dtype = Union[ExtensionDtype, Type[Union[str, float, int, complex, bool]]]
```

Comment:
```suggestion
Dtype = Union["ExtensionDtype", Type[Union[str, float, int, complex, bool]]]
```

Should be quoted. My mistake on earlier suggestion